capture-sdk/sdk/src/main/java/net/gini/android/capture/analysis/AnalysisScreenPresenter.java

```diff
@@ -57,9 +57,7 @@
 import static net.gini.android.capture.tracking.EventTrackingHelper.trackAnalysisScreenEvent;
 
 /**
- * Created by Alpar Szotyori on 08.05.2019.
- * <p>
- * Copyright (c) 2019 Gini GmbH.
+ * Internal use only
```

Comment:
[Request] We should keep the copyright section and add the `internal use only` below it in a separate javadoc like you did for AnalysisHintsAnimator.